client/src/panels/panel_declarations/results/IndicatorDisplayPanel.js

```diff
@@ -117,28 +119,34 @@ export default class IndicatorDisplayPanel extends React.Component {
   }
 
   render() {
-    const { id } = this.props;
+    const { id, subject, active_bubble_id } = this.props;
 
     const { loading } = this.state;
 
     const indicator = Indicator.lookup(id);
 
+    const panel_link = window.location.href.replace(
+      window.location.hash,
+      infograph_options_href_template(subject, active_bubble_id, {
```

Comment:
I think it's safe to just put "result" straight in, without the need of props. Sorry for not being clear on that and keep asking for changes!